src/tasks/importEvents.js

```diff
@@ -0,0 +1,102 @@
+const https = require('https');
+const request = require('request-promise-native');
+const { mobilizeAmericaToMongoDocument } = require('../transformers/event');
+const { setupDatabase } = require('../utils/connectToDatabase');
+
+function promotedOrganizationsUrl() {
+  return 'https://api.mobilize.us/v1/organizations/1316/promoted_organizations';
```

Comment:
Can we make these organization ID's a stage specific environment variable? That way the `dev` stage can point at the Mobilize America QA instance.

Reply:
Yup good idea